data/tutorials/en/001_a_first_hour_with_ocaml.md

```diff
@@ -10,7 +10,7 @@ date: 2021-08-06T17:11:00-00:00
 ---
 
 You may follow along with this tutorial with just a basic OCaml installation,
-as described in [Up and Running](up_and_running.html).
+as described in [Up and Running](https://v3.ocaml.org/tutorials/up-and-running-with-ocaml).
```

Comment:
Let's use an absolute path without the domain name, this will ensure that things don't break when we move `v3.ocaml.org` to `ocaml.org`:
```suggestion
as described in [Up and Running](/tutorials/up-and-running-with-ocaml).
```

Reply:
Great info & idea!